ui/src/App.less

```diff
@@ -42,6 +42,6 @@ a:active {
   text-decoration: underline;
 }
 
-.semi-icon {
+.semi-icon:not(.semi-tabs-bar .semi-tabs-tab .semi-icon) {
```

Comment:
This was causing trouble in the menu's tab bar (sorry, my mistake from earlier).